src/controllers/dlgprefcontroller.cpp

```diff
@@ -543,6 +546,25 @@ void DlgPrefController::slotPresetLoaded(ControllerPresetPointer preset) {
     }
 }
 
+void DlgPrefController::setupWarningLabel(ControllerPresetPointer preset) {
+    if (presetIsSupported(preset)) {
+        m_ui.groupBoxWarning->hide();
+    } else {
+        m_ui.groupBoxWarning->show();
+        m_ui.labelWarning->setText(tr("<font color='#BB0000'><b>If you use this preset your controller may not work correctly. "
```

Comment:
I think this should do more than warn the user. I think Mixxx should refuse to load the preset.

Reply:
Now the apply button, accept button, and all the buttons modifying the mapping are disabled. The controller is highlighted in yellow in the side-panel list and a warning appears above the apply button.
![screenshot 2019-01-02 at 01 29 13](https://user-images.githubusercontent.com/6429775/50577698-0a8fee00-0e2e-11e9-95ca-86f214395ae0.png)